docs/src/_contribute/ui.md

```diff
@@ -22,7 +22,7 @@ meltano init $PROJECT_NAME
 cd $PROJECT_NAME
 
 # Start the Meltano API and a production build of Meltano UI that you can ignore
-meltano ui
+FLASK_DEBUG=True meltano ui
```

Comment:
@WillDaSilva @pandemicsyn should we reccommend that this be done on a separate line?

```suggestion
export FLASK_DEBUG=True 

meltano ui
```

Reply:
Sure. That plus a short comment explaining explanation why would be ideal.

Reply:
It's on two separate lines now. I also fixed in the API dev documentation which had the same command on a single line (which is why I added it to a single line in the first place). Keep in mind we forfeit windows support by adding the `export` keyword, the rest of the script is windows compatible.